### DISCUSSION TITLE
Training PP-OCRv4 on digits. Help with config file

### DISCUSSION THREAD

**Discussion by mazatov** [PaddlePaddle/PaddleOCR#12790] [Q&A]
I'm trying to train PP-OCRv4 on digits dataset. I have used the same dataset to train PP-OCRv3 successfully and now am trying to train PP-OCRv4 to hopefully see improvement. 

I created a config file, analogous to what I used for PP-OCRv3. However, upon training, I get very subpar results (best accuracy = 50%, reached after 6 epochs out of 500). What can I be setting up wrong? 

I trained the same way I trained PP-OCRv4. 
`python tools/train.py -c configs/rec/PP-OCRv4/digit_PP-OCRv4_rec.yml -o Global.pretrained_model=pretrain_models/ch_PP-OCRv4_rec_train/student`

Here's the config file I used: 
```
Global:
  debug: false
  use_gpu: true
  epoch_num: 500
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/ppocr_v4_digit
  save_epoch_step: 3
  eval_batch_step:
  - 0
  - 400
  cal_metric_during_train: true
  pretrained_model: refactor
  checkpoints: null
  save_inference_dir: null
  use_visualdl: false
  infer_img: doc/imgs_words/ch/word_1.jpg
  character_dict_path: ppocr/utils/digit_dict.txt
  max_text_length: &max_text_length 4
  infer_mode: false
  use_space_char: true
  distributed: true
  save_res_path: ./output/rec/predicts_ppocrv4.txt
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.0005
    warmup_epoch: 5
  regularizer:
    name: L2
    factor: 3.0e-05
Architecture:
  model_type: rec
  algorithm: SVTR_LCNet
  Transform: null
  Backbone:
    name: PPLCNetV3
    scale: 0.95
  Head:
    name: MultiHead
    head_list:
    - CTCHead:
        Neck:
          name: svtr
          dims: 120
          depth: 2
          hidden_dims: 120
          kernel_size:
          - 1
          - 3
          use_guide: true
        Head:
          fc_decay: 1.0e-05
    - NRTRHead:
        nrtr_dim: 384
        max_text_length: 25
Loss:
  name: MultiLoss
  loss_config_list:
  - CTCLoss: null
  - NRTRLoss: null
PostProcess:
  name: CTCLabelDecode
Metric:
  name: RecMetric
  main_indicator: acc
  ignore_space: false
Train:
  dataset:
    name: SimpleDataSet
    ds_width: false
    data_dir: train_data/jersey_crops/
    ext_op_transform_idx: 1
    label_file_list:
    - train_data/jersey_crops/train_jersey.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - RecConAug:
        prob: 0.5
        ext_data_num: 2
        image_shape:
        - 48
        - 48
        - 3
        max_text_length: *max_text_length
    - RecAug:
        tia_prob: 0.8
        crop_prob: 0.8
        reverse_prob: 0.4
        noise_prob: 0.8
        jitter_prob: 0.8
        blur_prob: 0.8
        hsv_aug_prob: 0.8
    - MultiLabelEncode:
        gtc_encode: NRTRLabelEncode
    - RecResizeImg:
        image_shape: [3, 48, 48]        
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_gtc
        - length
        - valid_ratio
        

  loader:
    shuffle: true
    batch_size_per_card: 128
    drop_last: true
    num_workers: 4
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: train_data/jersey_crops/
    label_file_list:
    - train_data/jersey_crops/test_jersey.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - MultiLabelEncode:
        gtc_encode: NRTRLabelEncode
    - RecResizeImg:
        image_shape:
        - 3
        - 48
        - 48
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_gtc
        - length
        - valid_ratio
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 128
    num_workers: 4
profiler_options: null

```
